Polish/przygoda/miesnie/miesnie.md

```diff
@@ -0,0 +1,7 @@
+Siłujesz się ze ścianami przez kilka godzin, aż w końcu opadasz z sił.
+
+Co robisz?
+
+[Rozbijasz ściany siłą woli](../wola/wola.md)
+
+[Zaczynasz jeść ptasie mleczko](jedzenie/jedzenie.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.